### PR TITLE
Get cards from server

### PR DESCRIPTION
**Pull request by Zenater**
rewrite get cards, and fix Remark

### CODE REVIEW

File: src/components/cards/Card/Card.tsx [hunk 1]
```diff
@@ -26,17 +23,13 @@ export const Card = ({ title, date, userName, status }: PropsCardType) => {
...
-            {userName}
+            {author.name}
           </p>
         </div>
         <div className='flex flex-col text-center items-center gap-4'>
           <Link href='/testPage'>
             <Button>{t('start')}</Button>
           </Link>
-          <div>
-            <span className='text-text-primaryAlpha300'>Created: </span>
```
Comment (gitkoshelew) [inline src/components/cards/Card/Card.tsx:37]: why did you remove 'Created" ?

File: src/components/cards/Card/Card.tsx [hunk 2]
```diff
@@ -1,23 +1,20 @@
...
 type PropsCardType = {
   title: string;
-  userName: string;
-  date: string;
-  status: string | null;
+  author: AuthorType;
 };
 
-export const Card = ({ title, date, userName, status }: PropsCardType) => {
+export const Card = ({ title, author }: PropsCardType) => {
```
Comment (gitkoshelew) [inline src/components/cards/Card/Card.tsx]: you can destruct athor here also { title, author: {status, name} }

File: src/components/cards/Card/Card.tsx (hunk 1 above)
Comment (Zenater) [reply src/components/cards/Card/Card.tsx:37]: because this field is not in the data received from the server

{"id": 1,
"title": "You Dont know NodeJS",
"authorId": 7,
"author": {
"id": 7,
"name": "asdasd",
"surname": null,
"email": "sdad2@mail.ru",
"nickname": "asasdsaa",
"password": "$2b$10$UZ6KUgDmp3N2MsibblSikOaVFP0rQ2NcMNYlglnI4sU0e.RU/XSnq",
"status": null
},
"question": []
}

File: src/components/cards/Card/Card.tsx (hunk 2 above)
Comment (Zenater) [reply src/components/cards/Card/Card.tsx]: fixed

**Review by gitkoshelew — APPROVED** [review]

**Review by visiteur85 — APPROVED** [review]
